common/web/keyman-version/build.sh

```diff
@@ -52,17 +52,19 @@ if builder_has_action build; then
   # Generate index.ts
   echo "
     // Generated by common/web/keyman-version/build.sh
-    namespace KEYMAN_VERSION {
-      export const
-      VERSION = \"$VERSION\",
-      VERSION_RELEASE =\"$VERSION_RELEASE\",
-      VERSION_MAJOR = \"$VERSION_MAJOR\",
-      VERSION_MINOR = \"$VERSION_MINOR\",
-      VERSION_PATCH = \"$VERSION_PATCH\",
-      TIER =\"$TIER\",
-      VERSION_TAG = \"$VERSION_TAG\",
-      VERSION_WITH_TAG = \"$VERSION_WITH_TAG\",
-      VERSION_ENVIRONMENT = \"$VERSION_ENVIRONMENT\";
+    namespace com.keyman {
+      export class KEYMAN_VERSION {
+        static readonly VERSION = \"$VERSION\";
```

Comment:
This change set was necessary for two reasons:

1. to reflect what the `KEYMAN_VERSION` actually is - not a namespace, but an object.
2. By making TS recognize it as a class, we can make an alias declaration to its type, allowing the lexical model compiler components (etc) to still get solid type-declaration data.  (This was responsible for certain pre-rebase build failures.)

Sadly, VS Code's Intellisense will mention the namespaced form when hovering on `KEYMAN_VERSION`... but at least that's the only "loss."